docs/multiarch.rst

```diff
@@ -523,6 +524,51 @@ the API method ``create_prod_build`` specifying which platform a build
 is required for. If no platform is specified, no node selector will be
 used.
 
+Platform description
```

Comment:
Instead of using a prefix to identify platform specific configurations, what if we introduced a new configuration file? `osbs_platforms.yaml` ?

Not sure if this makes the system more complicated overall. Just mentioning it now because we are unlikely to change it later on.